awx/main/tests/functional/test_credential_migration.py

```diff
@@ -1,331 +0,0 @@
-import mock
```

Comment:
This was a pain to convert, and not really worth it given that this migration is cir. Tower 3.2